image_utils.py

```diff
@@ -92,7 +92,7 @@ def on_epoch_end(self, epoch, logs=None):
 
 # adapted from: https://stanford.edu/~shervine/blog/keras-how-to-generate-data-on-the-fly
 class ImagesAndMasksGenerator(Sequence):
-    def __init__(self, dataset_directory, rescale, target_size, batch_size, shuffle=True, seed=None, random_rotation=False):
+    def __init__(self, dataset_directory, rescale, target_size, batch_size, shuffle=False, seed=None, random_rotation=False):
```

Comment:
Should `training_data_shuffle_seed` from config be passed to here?

Reply:
nvm i guess, since I see the seed below in `train_segmentation_model.py`

Reply:
yep - it does. in line 70 in train_segmentation_model.py you'll see the training generator is constructed by:
```
    train_generator = ImagesAndMasksGenerator(
        Path(local_dataset_dir, train_config['dataset_id'], 'train').as_posix(),
        rescale=1./255,
        target_size=target_size,
        batch_size=batch_size,
        shuffle=True,
        random_rotation=train_config['data_augmentation']['random_90-degree_rotations'],
        seed=train_config['training_data_shuffle_seed'])
```